driver-core/src/test/java/com/datastax/driver/core/PrimitiveTypeSamples.java

```diff
@@ -56,6 +56,8 @@
                 .put(DataType.cint(), Integer.MAX_VALUE)
                 .put(DataType.text(), "text")
                 .put(DataType.timestamp(), new Date(872835240000L))
+                .put(DataType.date(), -2147467577)
```

Comment:
Was just looking at this, all the other tests seem to behaving well now with exception to when date is used in a UDT (UserTypesTest#testPrimitiveDatatypes):

    com.datastax.driver.core.exceptions.InvalidTypeException: Value o is of type 'org.apache.cassandra.db.marshal.SimpleDateType'
	  at com.datastax.driver.core.AbstractGettableByIndexData.checkType(AbstractGettableByIndexData.java:75)
	  at com.datastax.driver.core.AbstractData.setDateWithoutTime(AbstractData.java:173)
	  at com.datastax.driver.core.UserTypesTest.testPrimitiveDatatypes(UserTypesTest.java:293)

Not sure if this is an issue on the driver end or with the metadata coming back from Cassandra.  Looking into it, but thought I'd share while looking at it.